lib/runtime/wasmer/imports.go

```diff
@@ -526,8 +526,8 @@ func ext_crypto_sr25519_public_keys_version_1(context unsafe.Pointer, keyTypeID
 		return C.int64_t(ret)
 	}
 
-	if ks.Type() != crypto.Sr25519Type {
-		logger.Warn("[ext_crypto_ed25519_public_keys_version_1]", "name", id, "error", "keystore type is not ed25519")
+	if ks.Type() != crypto.Sr25519Type && ks.Type() != crypto.UnknownType {
+		logger.Warn("[ext_crypto_sr25519_public_keys_version_1]", "name", id, "error", "keystore type is not ed25519")
```

Comment:
Just need to update the error message for the correct key type

```suggestion
		logger.Warn("[ext_crypto_sr25519_public_keys_version_1]", "name", id, "error", "keystore type is not sr25519")
```

Reply:
Or if this is just checking for er25519, wouldn't this log if the key type is ed25519?